openapi/v3/device-sdk.yaml

```diff
@@ -578,6 +604,69 @@ paths:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
 
+  /profilescan:
+    parameters:
+      - $ref: '#/components/parameters/correlatedRequestHeader'
+    post:
+      description: Run the discovery request to generate a associated device profile for a specified device.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              allOf:
+                - $ref: '#/components/schemas/ProfileScanRequest'
+        required: true
+      responses:
+        '201':
+          description: "Created"
+          headers:
+            X-Correlation-ID:
+              $ref: '#/components/headers/correlatedResponseHeader'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BaseResponse'
+              example:
+                apiVersion: "v3"
+                requestId: "e6e8a2f4-eb14-4649-9e2b-175247911369"
+                statusCode: 201
+        '404':
```

Comment:
add 409 conflict
check if the profile name exists, if yes return 409 directly

Reply:
if the profile scan API is called with the same device name and the previous request is still running, return 409 as well with the clear explanation in the message.